GoogleDAIDemo/app/src/main/java/com/jwplayer/googledai/MainActivity.java

```diff
@@ -0,0 +1,165 @@
+package com.jwplayer.googledai;
+
+import android.os.Bundle;
+import android.os.Handler;
+import android.util.Log;
+import android.view.View;
+import android.widget.RelativeLayout;
+import androidx.appcompat.app.AppCompatActivity;
+import com.longtailvideo.jwplayer.JWPlayerView;
+import com.longtailvideo.jwplayer.configuration.PlayerConfig;
+import com.longtailvideo.jwplayer.events.AdErrorEvent;
+import com.longtailvideo.jwplayer.events.ReadyEvent;
+import com.longtailvideo.jwplayer.events.listeners.AdvertisingEvents;
+import com.longtailvideo.jwplayer.events.listeners.VideoPlayerEvents;
+import com.longtailvideo.jwplayer.media.ads.dai.ImaDaiSettings;
+import com.longtailvideo.jwplayer.media.playlists.PlaylistItem;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MainActivity extends AppCompatActivity
+        implements View.OnClickListener,
+        AdvertisingEvents.OnAdErrorListener,
+        VideoPlayerEvents.OnReadyListener,
+        JWPlayerNativeControls.OnControlsInteraction {
+
+    JWPlayerView mPlayerView;
+    JWPlayerNativeControls mNativePlayerControls;
+    RelativeLayout mTouchInterceptorView;
+
+    final Handler mHideControlsHandler = new Handler();
+    final long mDelayBeforeControlsAreHidden = 3000;
+
+    Runnable mHideControlsRunnable = new Runnable() {
+        @Override
+        public void run() {
+            mNativePlayerControls.setVisibility(View.INVISIBLE);
+            enableTouchInterceptor();
+        }
+    };
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        mPlayerView = findViewById(R.id.jwplayer);
+        mPlayerView.addOnReadyListener(this);
+
+        mNativePlayerControls = findViewById(R.id.player_control_view);
```

Comment:
@jlod9412 As above, let's use our normal controls instead of the native controls demo class.